pkg/purl/purl_test.go

```diff
@@ -181,6 +181,22 @@ func TestNewPackageURL(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "golang package with a local path",
+			typ:  ftypes.GoModule,
+			pkg: ftypes.Package{
+				Name:    "./private_repos/cnrm.googlesource.com/cnrm/",
```

Comment:
The local path in the name is not a valid PURL anyway, isn't it?

Reply:
I'm not sure if it's correct, but it is [valid](https://go.dev/play/p/AZnq_Lsit88).

Reply:
>A purl or package URL is an attempt to standardize existing approaches to reliably identify and locate software packages.
>A purl is a URL string used to identify and locate a software package in a mostly universal and uniform way across programing languages, package managers, packaging conventions, tools, APIs and databases.

https://github.com/package-url/purl-spec/tree/master#solution

I feel like it doesn't make sense to generate PURLs for local paths according to the definition because the PURL string cannot identify and locate the software package. Any ideas?

Reply:
Yes, I agree. Perhaps the change should be don't include the `purl` for local packages.

Reply:
Can we just skip generating PURLs for local packages?

Reply:
Done